tests/ocean/test_compute_flow.py

```diff
@@ -17,6 +17,16 @@
     mint_tokens_and_wait, get_resource_path, wait_for_ddo)
 
 
+def test_check_url():
+    check_url_endpoint = DataServiceProvider.build_check_url_endpoint()
+    result = DataServiceProvider.check_url(
+        'http://xkcd.com/349/info.0.json', check_url_endpoint
```

Comment:
I have uploaded that to our S3 buckets, so we can control it
```suggestion
        'https://s3.amazonaws.com/testfiles.oceanprotocol.com/info.0.json', check_url_endpoint
```